server/lib/currency.ts

```diff
@@ -130,6 +130,8 @@ export async function getFxRate(
   date: string | Date = 'latest',
 ): Promise<number> {
   debug(`getFxRate for ${date} ${fromCurrency} -> ${toCurrency}`);
+  fromCurrency = fromCurrency?.toUpperCase();
+  toCurrency = toCurrency?.toUpperCase();
```

Comment:
This is to make `getFxRate` more resilient in the future, in case there are other places where we pass it lowercase currencies.